osc_sdk/sdk.py

```diff
@@ -272,7 +274,10 @@ class FcuCall(ApiCall):
 
     def get_response(self, http_response):
         if http_response.status_code not in SUCCESS_CODES:
-            raise OscApiException(http_response)
+            fcu_exception = OscApiException(http_response)
+            fcu_exception.request_id = (
+            xmltodict.parse(http_response.content)['Response']['RequestID'])
```

Comment:
- Dans la code, la `requestId` est calculé dans `class OscApiException`
- `http_reponse` est passé à `OscApiException`
- pourquoi `request_id` est calculé separement ici, de `http_reponse`, et pas dans la même `class`, dans la même endroit ?

Reply:
moved to inside of class OscApiException